lib/tree.rb

```diff
@@ -1,25 +1,41 @@
 class NoApplesError < StandardError; end
 
-class AppleTree
-  attr_#fill_in :height, :age, :apples, :alive
+class Tree
+  attr_reader :height, :age
+  MAXIMUM_AGE = 100
+  MAXIMUM_FRUIT_DIAMETER = 10
 
   def initialize
+    @height = 0.5
+    @age = 0
+    @apples = []
   end
 
   def age!
+    @age += 1
+    @height += 0.5 unless height >= 30
+    add_apples
   end
 
   def add_apples
+    if @age >= 3
+      rand(@age / 2...@age).times do
```

Comment:
I was hoping to get some clarity on `rand(@age / 2...@age)` because it doesn't make much sense to me.

Looking at it again, the problem is an integer divided by a range. I've never seen that before. What is that supposed to mean? Is that really doing what you think it is doing?